tilecloud/store/s3.py

```diff
@@ -85,7 +85,7 @@ def put_one(self, tile):
 
 
 def _get_status(s3_client_exception):
-    return int(s3_client_exception.response['Error']['Code'])
+    return s3_client_exception.response['ResponseMetadata']['HTTPStatusCode']
```

Comment:
Watch out, it tends to be stored by boto3 as a string. So please put a `int()` around it.

Reply:
Here we have an int ...

Reply:
Or maybe not... why did you change that? I have code in c2cwsgiutils that is similar to the previous version. Maybe I should change it also?

Reply:
OK